lib/core.js

```diff
@@ -76,43 +76,15 @@ export function Meta(name, content) {
 /*
   Func: loadDoc
 	Param: url to file
-	Return: return string of file
+	Return: return object of file
 */
-export function loadDoc(url) { 
-  return new Promise(function (resolve, reject) {
-    
-    let xhr = new XMLHttpRequest();
-
-    // Setup our listener to process compeleted requests
-    xhr.onreadystatechange = function () {
-      // Only run if the request is complete
-      if (xhr.readyState !== 4) return;
-
-      // Process our return data
-      // status = 0 for local files
-      if ((xhr.status == 0 || xhr.status >= 200) && xhr.status < 300) {
-        resolve(xhr.responseText)
-
-      // Request Fails
-      } else {
-        reject({
-            status: this.status,
-            statusText: xhr.statusText
-        });
-      }
-    };
-    
-    xhr.onerror = function () {
-        reject({
-            status: this.status,
-            statusText: xhr.statusText
-        });
-    };
-
-    xhr.open("GET", url, true);
-    xhr.send();
-
-  });
+export async function loadDoc(url) { 
+  try {
+    const response = await fetch(url);
+    return await response.json();
```

Comment:
Needed to change this back to 
`return await response.text()`
otherwise will not parse the markdown appropriately.